### DISCUSSION TITLE
Github copilot not singed in

### DISCUSSION THREAD

**Discussion by Dmtr1i** [community/community#120556] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Yesterday the download worked fine, but today I can't download. I get an error NotSignedIn

**Comment by ayim** [comment]
Which IDE are you using?

**Reply by Dmtr1i** [reply]
Microsoft Visual Studio 2022

**Comment by davecheney** [comment, marked as answer]
@Dmtr1i there is no copilot licence associated with this GitHub user.

**Reply by Dmtr1i** [reply]
Strange, everything worked before. How can I get it?

**Reply by davecheney** [reply]
@Dmtr1i the account @Dmtr1i has never had a copilot subscription. Were you possibly using a different GitHub account?

**Reply by Dmtr1i** [reply]
I used this account and had no problems. Can I upload my projects from MS Visual Studio without a copilot subscription?

**Reply by davecheney** [reply]
The account @Dmtr1i has no GitHub Copilot subscription. This only affects the use of GitHub Copilot, not other * Copilot branded products or other products from the Visual Studio family.

**Reply by Dmtr1i** [reply]
My mistake was that when installing the components, I accidentally installed Github Copilot and did not notice it. After uninstalling everything works fine. Thanks for the help!

**Comment by robinhood074** [comment]
@Dmtr1i  As far as I am aware, this GitHub user does not have a copilot license.
